__tests__/api.test.js

```diff
@@ -167,3 +167,89 @@ describe('GET/api/articles/:article_id/comments', () => {
         })
     });
 });
+
+
+describe('POST /api/articles/:article_id/comments', () => {
+    it('201: responds with a status of 201', () => {
+        const newComment = {
+            username: "butter_bridge",
+            body: "Hello"
+        };
+
+        return request(app).post('/api/articles/4/comments').send(newComment)
+        .expect(201);
+    });
+    
+    it('201: responds with a comment object that has been sent', () => {
+        const newComment = {
+            username: "butter_bridge",
+            body: "Hello"
+        };
+
+        return request(app).post('/api/articles/4/comments').send(newComment)
+        .expect(201)
+        .then((response) => {
+            const {comment} = response.body;
+            expect(comment.author).toEqual(newComment.username);
+            expect(comment.body).toEqual(newComment.body);
+            expect(comment.votes).toEqual(0);
+            expect(comment.created_at).not.toBe("");
+            expect(comment.comment_id).not.toBe(0);
+        })
```

Comment:
Please write another test where the extra properties are ignored. For example when posting with:

 const newComment = {
           username: 'butter_bridge',
           body: 'This is a test comment.', 
           extraProperty: "someValue"
       };